roxie/ccd/ccdserver.cpp

```diff
@@ -21841,6 +21843,7 @@ class CRoxieServerDiskReadBaseActivity : public CRoxieServerActivity, implements
     bool isLocal;
     bool forceRemote;
     bool isGrouped = false;
+    bool isCodeSigned = false;
```

Comment:
unfortunately this pushes up the size of all disk read activities by 8 bytes.

@richardkchapman would this be better to not store here, and use factory->isCodeSigned() instead?

Reply:
I suspect there are quite a few flags that would be better off stored in the factory rather than being copied into the activity (especially as we seem to always pass as parameters to the constructor too).

Reply:
I'm using the flag from the factory.  It's made the code some much simpler.